### PR TITLE
fix: 404 page background gradient

### PR DESCRIPTION
**Pull request by tokitou-san**

### CODE REVIEW

File: src/lib/components/pages/anime_info/error.svelte [hunk 1]
```diff
@@ -2,11 +2,10 @@
...
 </script>
 
-<section class="grid h-full grid-cols-5 items-end justify-between md:px-[5vw]">
-    <radial-gradient
-        class="pointer-events-none absolute inset-0 z-10 md:translate-x-[30vw]"
-        style="background-image: radial-gradient(circle at center, rgba(117, 105, 225, 0.1) 0%, transparent 100%);"
-    />
+<section
+    class="relative grid h-full grid-cols-5 items-end justify-between md:px-[5vw]"
+    style="background-image: linear-gradient(to top left, rgba(117, 105, 225, 0.1) 0%, transparent 50%);"
```
Comment (baseplate-admin) [inline src/lib/components/pages/anime_info/error.svelte:7]: Make it center?

**Comment by baseplate-admin** [comment]
@tokitou-san, LMK if this makes better sense :)

**Comment by tokitou-san** [comment]
Screenshot from 2023-07-14 22-42-32

Looks good in mobile?
and it seems like `40vw` is the only perfect value for both.... but isnt it looks small in mobile?

It looks really good in desktop :P
what about add a gradient from bottom like I did before for mobile only?
gradient has height issues with mobile....

Screenshot from 2023-07-14 22-48-54

See? Lemme see if there is any other fix for this :P

**Comment by baseplate-admin** [comment]
Fixed

**Comment by tokitou-san** [comment]
Screenshot from 2023-07-15 10-18-21

It still has height issue, see?
needa move gradient element outside character element
or adjust height of image gradient

You on this?

**Comment by baseplate-admin** [comment]
fixed

File: src/lib/components/pages/anime_info/error.svelte [hunk 2]
```diff
@@ -5,10 +5,10 @@
     const mapping = _.sample([{ image: "/images/characters/eliane.png", alt: "Elaine" }]);
 </script>
 
-<section class="relative grid h-full grid-cols-5 items-end justify-between md:px-[5vw]">
-    <error-context class="col-span-12 flex flex-col items-center leading-none md:col-span-3 md:mb-[12vw] md:items-start md:gap-[1vw]">
+<section class="relative flex h-full grid-cols-5 flex-col items-center justify-end gap-20 md:grid md:items-end md:gap-0 md:px-[5vw]">
+    <error-context class="col-span-5 flex flex-col items-center leading-none md:col-span-3 md:mb-[13vw] md:items-start md:gap-[1vw]">
         <status-code class="text-7xl font-bold md:text-[7vw]">
-            {#each "404".split("") as number}
+            {#each Array.from("404") as number}
```
Comment (baseplate-admin) [inline src/lib/components/pages/anime_info/error.svelte:11]: Not a big fan

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

**Comment by tokitou-san** [comment]
@baseplate-admin Now its ready to merge...

**Pull request merged**